Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerGrainStore.cs

```diff
@@ -38,6 +38,28 @@ public class SqlServerGrainStore : SqlServerBaseStore, IGrainStore
             return grain.ToModel();
         }
 
+        public Task<IEnumerable<Grain>> GetAll()
+        {
+            var grains = AuthorizationDbContext.Grains
+                .Include(g => g.SecurableItems)
+                .Where(g => !g.IsDeleted);
+
+            foreach (var grain in grains)
+            {
+                foreach (var securableItem in grain.SecurableItems)
+                {
+                    LoadChildrenRecursive(securableItem);

```

Comment:
We may need to update this function to filter our deleted securables.

Reply:
This function is used within our securableItems code as well.  I would need to create something that is a separate function.

Reply:
Perhaps that's a bug in the existing code? Is there ever a case where we need to return deleted ones?

Reply:
That is what I was thinking, we can just look to see what is using this function and determine if the current usages need deleted items or not.

Reply:
Tracing up the call stack, it's used in `GetRequiredWriteClaims(GrainService grainService, string grain)` in `FabricModule` and when adding a client. I'm thinking they *should* be filtered out for these scenarios.

Reply:
Agreed.

Reply:
kk, i will filter deleted stuff.  I went back to look for the other usage, and i cant find it.  Thought Securable items was working the same way. 

Ah well, added this code and if i run into this for securable items, i will change it.